projects/Shapiro Andrey/main.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+app = FastAPI()
+
+class Car(BaseModel):
+    id: int
+    brand: str
+    model: str
+    year: int
+
+cars_db = []
+current_id = 1
+
+@app.post("/cars/", response_model=Car)
+def create_car(car: Car):
```

Comment:
We have to separate Car models for adding a new car and getting info on an existing one. There is no need to provide an ID for creating a new Car.